### ISSUE TITLE
Document Negative Weights and other Angular Grid Peculiarities

### ISSUE THREAD

**Issue by PaulWAyers** [theochem/grid#158]
I'm attaching a script and the resulting plot with errors on simple density integrals over the whole molecular grid, as function of the angular grid size. The radial grid is always the same (see script). Vertical lines correspond to the grids with negative weights. The three cases do not pop out as extremely problematic, so a warning may be too pedantic.

That said, the result for `he2_ghost_psi4_1.0.molden` is troubling. After some trial and error, this turned out to be related to the fact that the Bragg radius for Helium is set to `Nan`. This still results in a grid (with all finite weights) without raising an exception nor a warning. I'll open a separate issue for it.

@PaulWAyers Do you think the non-smooth trend of the error as function of grid size is fine? I would think so because one can always get a small error by chance.

[lebcheck.zip](https://github.com/theochem/grid/files/4994127/lebcheck.zip)

![lebcheck](https://user-images.githubusercontent.com/99431/88786254-98f75d80-d192-11ea-98a8-dbb08ac16ecd.png)

_Originally posted by @tovrstra in https://github.com/theochem/grid/issues/99#issuecomment-665574885_

**Comment by PaulWAyers** [comment]
A good choice is to use a more modern set of covalent radii (defined up to Z=96)
    https://pubs.rsc.org/en/content/articlelanding/2008/DT/b801115j
This should be the default radii I think, as the methodology is Bragg-like (based on crystal structures)

@marco-2023 can you add this to the file 
    https://github.com/theochem/grid/blame/master/src/grid/utils.py

When a NaN is encountered, we switch to the element with one smaller Z. That is OK, but we should raise a warning that this happened.

**Comment by marco-2023** [comment]
@PaulWAyers 
Added the new set of covalent radii.
I do not have push permissions to https://github.com/theochem/grid I created a fork, pushed the changes and created a pull request #160. Some tests failed?

**Comment by PaulWAyers** [comment]
Thanks. Yep, creating a pull request is (almost always) the right thing to do.

**Comment by PaulWAyers** [comment]
We should make a list of Lebedev grids with negative weights, and also specify where angular grids are nested. this should be included in the documentation.

**Comment by Ali-Tehrani** [comment]
Lebedev grids with negative weights are now reported as warning from commit 6b93f31

The nested grids are now added as .csv files and the distances between two consequent angular grids are reported as a table on the [website](https://grid.qcdevs.org/conventions.html#nested-grids).

**Comment by Ali-Tehrani** [comment]
I didn't do any tricks in order to get the distance between the grid points.  I had to increase the memory to 50 GB for symmetric spherical t-design grids.    I was thinking of doing K-d tree or doing a chunked memory approach to calculate pairwise distances from sklearn [pairwise distanced chunked](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.pairwise_distances_chunked.html) but increasing memory was the easiest path.

```python
dist = cdist(grid.points, grid2.points)
minimum_dist = np.min(dist, axis=1)
print(np.mean(minimum_dist), np.std(minimum_dist), np.max(minimum_dist))
```
I added a star next to Lebedev grids with negative weights
